### PR TITLE
add body api safari, safari mobile, chrome android support

### PR DESCRIPTION
**Pull request by AllanFly120**
**Summary**:
Update Safari, Safari iOS, chrome mobile support for [Body API](https://developer.mozilla.org/en-US/docs/Web/API/Body) 

**Data**:
* Apple DN: https://developer.apple.com/documentation/webkitjs/fetchbody
* Chromium commit: https://github.com/chromium/chromium/commit/651abb3f560b95d7c9748ed7b7ea1470af2a2a76, introduced in Chrome: 45.0.2454.101

**Tests**
Testing with the Page 👉: https://software.hixie.ch/utilities/js/live-dom-viewer/?saved=7648
macOS Sierra, Safari 10.2 screenshot
Screen Shot 2020-01-13 at 5 27 43 PM
iOS 12.1.4, iPad Air 2 screenshot
Screen Shot 2020-01-14 at 12 02 18 AM
Android 8.0.0, Motorola Moto G6, chrome screenshot:
Screen Shot 2020-01-14 at 12 23 46 AM

A checklist to help your pull request get merged faster:
- [x] Summarize your changes
- [x] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)
- [x] Data: if you tested something, describe how you tested with details like browser and version
- [x] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [ ] Link to related issues or pull requests, if any

### CODE REVIEW

File: api/Body.json [hunk 1]
```diff
@@ -107,7 +107,7 @@
               }
             ],
             "chrome_android": {
-              "version_added": false
+              "version_added": true
```
Comment (ddbeck) [inline api/Body.json]:
```suggestion
              "version_added": "42"
```

File: api/Body.json [hunk 2]
```diff
@@ -199,7 +199,7 @@
...
```
Comment (ddbeck) [inline api/Body.json]:
```suggestion
              "version_added": "42"
```

File: api/Body.json [hunk 3]
```diff
@@ -524,7 +524,7 @@
...
```
Comment (ddbeck) [inline api/Body.json]:
```suggestion
              "version_added": "42"
```

File: api/Body.json [hunk 4]
```diff
@@ -616,7 +616,7 @@
...
```
Comment (ddbeck) [inline api/Body.json]:
```suggestion
              "version_added": "42"
```

**Comment by AllanFly120** [comment]
@ddbeck Thank you for your review. Is there any other updates required here?

**Pull request merged**